tests/async_requests/executor.py

```diff
@@ -1,33 +1,102 @@
 from django.test import TestCase
 
 from apps.async_requests.factory import Factory
-from mocks.command import MockCommand
+from mocks.command import MockCommand, SecondMockCommand
 
 
+# TODO user unittest.TestCase instead because test doesn't involve DB
```

Comment:
Does it make a difference?

Reply:
speed
https://docs.djangoproject.com/en/3.2/topics/testing/overview/#writing-tests warning section

Reply:
This line should be removed